jobs/jobsprotectedts: TestSchedulesProtectedTimestamp failed

jobs/jobsprotectedts.TestSchedulesProtectedTimestamp [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3675240&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3675240&tab=artifacts#/) on master @ [05d36621ae805e644d98ef6b35d453ae16255ccc](https://github.com/cockroachdb/cockroach/commits/05d36621ae805e644d98ef6b35d453ae16255ccc):


```
I211103 06:58:48.502514 80716 server/node.go:499  [n1] 95  started with engine type 2
I211103 06:58:48.502630 80716 server/node.go:501  [n1] 96  started with attributes []
I211103 06:58:48.502916 80716 1@server/server.go:1688  [n1] 97  starting https server at 127.0.0.1:38519 (use: 127.0.0.1:38519)
I211103 06:58:48.503060 80716 1@server/server.go:1693  [n1] 98  starting postgres server at 127.0.0.1:44539 (use: 127.0.0.1:44539)
I211103 06:58:48.503239 80716 1@server/server.go:1695  [n1] 99  starting grpc server at 127.0.0.1:35863
I211103 06:58:48.503301 80716 1@server/server.go:1696  [n1] 100  advertising CockroachDB node at 127.0.0.1:35863
W211103 06:58:48.528490 82971 kv/range_lookup.go:243  [n1,rangefeed=table-stats-cache,range-lookup=/Meta2/Table/20/NULL] 101  range lookup of key /Meta2/Table/20/NULL found only non-matching ranges []; retrying
W211103 06:58:48.537519 82966 kv/range_lookup.go:243  [n1,rangefeed=table-stats-cache,range-lookup=/Table/20] 102  range lookup of key /Table/20 found only non-matching ranges []; retrying
I211103 06:58:48.559428 80716 1@util/log/event_log.go:32  [n1] 103 ={"Timestamp":1635922728559406002,"EventType":"node_join","NodeID":1,"StartedAt":1635922728486479793,"LastUp":1635922728486479793}
I211103 06:58:48.559567 80716 sql/sqlliveness/slinstance/slinstance.go:296  [n1] 104  starting SQL liveness instance
I211103 06:58:48.564413 86822 sql/sqlliveness/slstorage/slstorage.go:397  [n1] 105  inserted sqlliveness session c06b7258ad28434782a8d33169c0c71e
I211103 06:58:48.564505 86822 sql/sqlliveness/slinstance/slinstance.go:183  [n1] 106  created new SQL liveness session c06b7258ad28434782a8d33169c0c71e
I211103 06:58:48.565300 88375 sql/temporary_schema.go:508  [n1] 107  running temporary object cleanup background job
I211103 06:58:48.565478 88447 sql/sqlstats/persistedsqlstats/provider.go:138  [n1] 108  starting sql-stats-worker with initial delay: 51m36.213603679s
I211103 06:58:48.567318 88375 sql/temporary_schema.go:552  [n1] 109  found 0 temporary schemas
I211103 06:58:48.567557 88375 sql/temporary_schema.go:555  [n1] 110  early exiting temporary schema cleaner as no temporary schemas were found
I211103 06:58:48.567618 88375 sql/temporary_schema.go:556  [n1] 111  completed temporary object cleanup job
I211103 06:58:48.567662 88375 sql/temporary_schema.go:639  [n1] 112  temporary object cleaner next scheduled to run at 2021-11-03 07:28:48.565221589 +0000 UTC
I211103 06:58:48.596303 86077 kv/kvserver/replica_rangefeed.go:657  [n1,rangefeed=table-stats-cache,s1,r16/1:/Table/2{0-1}] 113  RangeFeed closed timestamp is empty
I211103 06:58:48.716603 96987 util/log/event_log.go:32  [n1,intExec=optInToDiagnosticsStatReporting] 114 ={"Timestamp":1635922728637506968,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"diagnostics.reporting.enabled\" = true","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-optInToDiagnosticsStatReporting","SettingName":"diagnostics.reporting.enabled","Value":"true"}
I211103 06:58:48.738474 104005 migration/migrationmanager/manager.go:110  [n1,intExec=set-setting,migration-mgr] 115  no need to migrate, cluster already at newest version
I211103 06:58:48.746536 104005 util/log/event_log.go:32  [n1,intExec=set-setting] 116 ={"Timestamp":1635922728731265764,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING version = $1","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-set-setting","PlaceholderValues":["'21.2-4'"],"SettingName":"version","Value":"21.2-4"}
I211103 06:58:48.816724 113922 util/log/event_log.go:32  [n1,intExec=initializeClusterSecret] 117 ={"Timestamp":1635922728803267470,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"cluster.secret\" = gen_random_uuid()::STRING","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-initializeClusterSecret","SettingName":"cluster.secret","Value":"0eded450-76a5-4bf4-a902-fa4525cdf2da"}
I211103 06:58:48.850557 116656 5@util/log/event_log.go:32  [n1,intExec=create-default-DB] 118 ={"Timestamp":1635922728823368620,"EventType":"create_database","Statement":"CREATE DATABASE IF NOT EXISTS defaultdb","Tag":"CREATE DATABASE","User":"root","DescriptorID":50,"ApplicationName":"$ internal-create-default-DB","DatabaseName":"defaultdb"}
I211103 06:58:48.865625 84555 3@vendor/github.com/cockroachdb/pebble/event.go:664  [n1,pebble,s1] 119  [JOB 3] WAL created 000004
I211103 06:58:48.872027 119234 5@util/log/event_log.go:32  [n1,intExec=create-default-DB] 120 ={"Timestamp":1635922728851472927,"EventType":"create_database","Statement":"CREATE DATABASE IF NOT EXISTS postgres","Tag":"CREATE DATABASE","User":"root","DescriptorID":51,"ApplicationName":"$ internal-create-default-DB","DatabaseName":"postgres"}
I211103 06:58:48.943720 80716 server/server_sql.go:1107  [n1] 121  done ensuring all necessary startup migrations have run
I211103 06:58:48.944409 129368 jobs/job_scheduler.go:357  [n1] 122  waiting 2m0s before scheduled jobs daemon start
I211103 06:58:48.957614 129374 server/auto_upgrade.go:55  [n1] 123  no need to upgrade, cluster already at the newest version
I211103 06:58:48.979904 80716 1@server/server.go:2345  [n1] 124  serving sql connections
I211103 06:58:48.980826 80716 testutils/testcluster/testcluster.go:1158  [-] 125  WaitForFullReplication
I211103 06:58:48.980885 80716 testutils/testcluster/testcluster.go:1162  [-] 126  WaitForFullReplication took: 418ns
I211103 06:58:48.980996 80716 2@rpc/context.go:1045  [n1] 127  dialing n1: 127.0.0.1:35863 (default)
I211103 06:58:49.026505 131028 util/log/event_log.go:32  [n1,client=127.0.0.1:43198,hostssl,user=root] 128 ={"Timestamp":1635922729010680047,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"kv.protectedts.reconciliation.interval\" = '1ms'","Tag":"SET CLUSTER SETTING","User":"root","SettingName":"kv.protectedts.reconciliation.interval","Value":"00:00:00.001"}
    jobs_protected_ts_test.go:203: 
        	Error Trace:	jobs_protected_ts_test.go:203
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 0
        	Test:       	TestSchedulesProtectedTimestamp
I211103 06:58:49.132727 80716 testutils/testcluster/testcluster.go:106  [-] 129  TestCluster quiescing nodes
I211103 06:58:49.132929 81283 2@rpc/context.go:688  [n1] 130  closing [{n1: 127.0.0.1:35863 (default)}]
W211103 06:58:49.133057 86822 sql/sqlliveness/slinstance/slinstance.go:220  [n1] 131  exiting heartbeat loop
I211103 06:58:49.133173 81283 2@rpc/context.go:688  [n1] 132  closing [{n0: 127.0.0.1:35863 (default)}]
I211103 06:58:49.133115 130760 2@rpc/context.go:688  [n1] 133  closing [{n1: 127.0.0.1:35863 (default)} {n0: 127.0.0.1:35863 (default)}]
W211103 06:58:49.133224 96046 jobs/registry.go:840  [-] 134  canceling all adopted jobs due to stopper quiescing
W211103 06:58:49.133286 141992 kv/kvserver/intentresolver/intent_resolver.go:790  [-] 135  failed to gc transaction record: could not GC completed transaction anchored at /Table/37/1/707304911934783489/0: node unavailable; try another peer
W211103 06:58:49.137776 86738 kv/txn.go:693  [n1] 136  failure aborting transaction: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer; abort caused by: failed to release record 1d853d6d-1d82-4751-837b-5b576f31c2a8: protectedts-Release: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
E211103 06:58:49.137899 86738 kv/kvserver/protectedts/ptreconcile/reconciler.go:179  [n1] 137  failed to reconcile protected timestamp with id 1d853d6d-1d82-4751-837b-5b576f31c2a8: failed to release record 1d853d6d-1d82-4751-837b-5b576f31c2a8: protectedts-Release: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
--- FAIL: TestSchedulesProtectedTimestamp (0.74s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=deadlock

- GOFLAGS=-parallel=4
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSchedulesProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

jobs/jobsprotectedts.TestSchedulesProtectedTimestamp [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3675241&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3675241&tab=artifacts#/) on master @ [05d36621ae805e644d98ef6b35d453ae16255ccc](https://github.com/cockroachdb/cockroach/commits/05d36621ae805e644d98ef6b35d453ae16255ccc):


```
I211103 06:59:40.630156 1557 1@server/server.go:1563  [n1] 95  node connected via gossip
I211103 06:59:40.630470 1462 server/node.go:422  [n1] 96  initialized store s1
I211103 06:59:40.630830 1462 kv/kvserver/stores.go:250  [n1] 97  read 0 node addresses from persistent storage
I211103 06:59:40.631535 1462 server/node.go:499  [n1] 98  started with engine type 2
I211103 06:59:40.631722 1462 server/node.go:501  [n1] 99  started with attributes []
I211103 06:59:40.632921 1462 1@server/server.go:1688  [n1] 100  starting https server at 127.0.0.1:42969 (use: 127.0.0.1:42969)
I211103 06:59:40.633157 1462 1@server/server.go:1693  [n1] 101  starting postgres server at 127.0.0.1:44595 (use: 127.0.0.1:44595)
I211103 06:59:40.633424 1462 1@server/server.go:1695  [n1] 102  starting grpc server at 127.0.0.1:40719
I211103 06:59:40.633548 1462 1@server/server.go:1696  [n1] 103  advertising CockroachDB node at 127.0.0.1:40719
I211103 06:59:40.672671 1462 1@util/log/event_log.go:32  [n1] 104 ={"Timestamp":1635922780672657091,"EventType":"node_join","NodeID":1,"StartedAt":1635922780599964194,"LastUp":1635922780599964194}
I211103 06:59:40.673108 1462 sql/sqlliveness/slinstance/slinstance.go:296  [n1] 105  starting SQL liveness instance
I211103 06:59:40.682182 1676 sql/sqlliveness/slstorage/slstorage.go:397  [n1] 106  inserted sqlliveness session b1d6aa6caf9e4b1387121bf7de0dd3e2
I211103 06:59:40.682588 1676 sql/sqlliveness/slinstance/slinstance.go:183  [n1] 107  created new SQL liveness session b1d6aa6caf9e4b1387121bf7de0dd3e2
I211103 06:59:40.683742 1681 sql/temporary_schema.go:508  [n1] 108  running temporary object cleanup background job
I211103 06:59:40.684258 1734 sql/sqlstats/persistedsqlstats/provider.go:138  [n1] 109  starting sql-stats-worker with initial delay: 1h3m50.947420026s
I211103 06:59:40.687100 1681 sql/temporary_schema.go:552  [n1] 110  found 0 temporary schemas
I211103 06:59:40.687279 1681 sql/temporary_schema.go:555  [n1] 111  early exiting temporary schema cleaner as no temporary schemas were found
I211103 06:59:40.687479 1681 sql/temporary_schema.go:556  [n1] 112  completed temporary object cleanup job
I211103 06:59:40.687638 1681 sql/temporary_schema.go:639  [n1] 113  temporary object cleaner next scheduled to run at 2021-11-03 07:29:40.683621755 +0000 UTC
W211103 06:59:40.691968 1555 kv/range_lookup.go:243  [n1,rangefeed=table-stats-cache,range-lookup=/Meta2/Table/20/NULL] 114  range lookup of key /Meta2/Table/20/NULL found only non-matching ranges []; retrying
W211103 06:59:40.697795 1554 kv/range_lookup.go:243  [n1,rangefeed=table-stats-cache,range-lookup=/Table/20] 115  range lookup of key /Table/20 found only non-matching ranges []; retrying
I211103 06:59:40.709744 1738 kv/kvserver/replica_rangefeed.go:657  [n1,rangefeed=table-stats-cache,s1,r16/1:/Table/2{0-1}] 116  RangeFeed closed timestamp is empty
I211103 06:59:40.804436 1877 util/log/event_log.go:32  [n1,intExec=optInToDiagnosticsStatReporting] 117 ={"Timestamp":1635922780755955555,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"diagnostics.reporting.enabled\" = true","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-optInToDiagnosticsStatReporting","SettingName":"diagnostics.reporting.enabled","Value":"true"}
I211103 06:59:40.832367 2039 migration/migrationmanager/manager.go:110  [n1,intExec=set-setting,migration-mgr] 118  no need to migrate, cluster already at newest version
I211103 06:59:40.842482 2039 util/log/event_log.go:32  [n1,intExec=set-setting] 119 ={"Timestamp":1635922780823573506,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING version = $1","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-set-setting","PlaceholderValues":["'21.2-4'"],"SettingName":"version","Value":"21.2-4"}
I211103 06:59:40.928388 2308 util/log/event_log.go:32  [n1,intExec=initializeClusterSecret] 120 ={"Timestamp":1635922780909281401,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"cluster.secret\" = gen_random_uuid()::STRING","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-initializeClusterSecret","SettingName":"cluster.secret","Value":"44b8da63-1fa1-4585-bd9a-3cb88c74051c"}
I211103 06:59:40.961726 2403 5@util/log/event_log.go:32  [n1,intExec=create-default-DB] 121 ={"Timestamp":1635922780941333754,"EventType":"create_database","Statement":"CREATE DATABASE IF NOT EXISTS defaultdb","Tag":"CREATE DATABASE","User":"root","DescriptorID":50,"ApplicationName":"$ internal-create-default-DB","DatabaseName":"defaultdb"}
I211103 06:59:40.979318 1618 3@vendor/github.com/cockroachdb/pebble/event.go:664  [n1,pebble,s1] 122  [JOB 3] WAL created 000004
I211103 06:59:40.987267 2362 5@util/log/event_log.go:32  [n1,intExec=create-default-DB] 123 ={"Timestamp":1635922780963475612,"EventType":"create_database","Statement":"CREATE DATABASE IF NOT EXISTS postgres","Tag":"CREATE DATABASE","User":"root","DescriptorID":51,"ApplicationName":"$ internal-create-default-DB","DatabaseName":"postgres"}
I211103 06:59:41.095291 1462 server/server_sql.go:1107  [n1] 124  done ensuring all necessary startup migrations have run
I211103 06:59:41.095741 2519 jobs/job_scheduler.go:357  [n1] 125  waiting 3m0s before scheduled jobs daemon start
I211103 06:59:41.109588 2521 server/auto_upgrade.go:55  [n1] 126  no need to upgrade, cluster already at the newest version
I211103 06:59:41.468884 1462 1@server/server.go:2345  [n1] 127  serving sql connections
I211103 06:59:41.473474 1462 testutils/testcluster/testcluster.go:1158  [-] 128  WaitForFullReplication
I211103 06:59:41.473594 1462 testutils/testcluster/testcluster.go:1162  [-] 129  WaitForFullReplication took: 885ns
I211103 06:59:41.473763 1462 2@rpc/context.go:1045  [n1] 130  dialing n1: 127.0.0.1:40719 (default)
I211103 06:59:41.553630 2603 util/log/event_log.go:32  [n1,client=127.0.0.1:42100,hostssl,user=root] 131 ={"Timestamp":1635922781531775305,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"kv.protectedts.reconciliation.interval\" = '1ms'","Tag":"SET CLUSTER SETTING","User":"root","SettingName":"kv.protectedts.reconciliation.interval","Value":"00:00:00.001"}
    jobs_protected_ts_test.go:203: 
        	Error Trace:	jobs_protected_ts_test.go:203
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 0
        	Test:       	TestSchedulesProtectedTimestamp
I211103 06:59:41.848251 1462 testutils/testcluster/testcluster.go:106  [-] 132  TestCluster quiescing nodes
W211103 06:59:41.848910 1838 jobs/registry.go:840  [-] 133  canceling all adopted jobs due to stopper quiescing
I211103 06:59:41.849078 1682 kv/kvclient/kvcoord/transport_race.go:107  [-] 134  transport race promotion: ran 37 iterations on up to 325 requests
W211103 06:59:41.849156 2806 kv/kvserver/intentresolver/intent_resolver.go:790  [-] 135  failed to gc transaction record: could not GC completed transaction anchored at /Table/37/1/707305084187836417/0: node unavailable; try another peer
I211103 06:59:41.849489 2650 2@rpc/context.go:688  [n1] 136  closing [{n1: 127.0.0.1:40719 (default)} {n0: 127.0.0.1:40719 (default)}]
W211103 06:59:41.852611 1676 sql/sqlliveness/slinstance/slinstance.go:220  [n1] 137  exiting heartbeat loop
--- FAIL: TestSchedulesProtectedTimestamp (1.53s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSchedulesProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

jobs/jobsprotectedts.TestSchedulesProtectedTimestamp [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3674493&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3674493&tab=artifacts#/) on master @ [05d36621ae805e644d98ef6b35d453ae16255ccc](https://github.com/cockroachdb/cockroach/commits/05d36621ae805e644d98ef6b35d453ae16255ccc):


```
I211103 06:13:22.246922 1383 server/node.go:499  [n1] 98  started with engine type 2
I211103 06:13:22.247079 1383 server/node.go:501  [n1] 99  started with attributes []
I211103 06:13:22.248201 1383 1@server/server.go:1688  [n1] 100  starting https server at 127.0.0.1:41321 (use: 127.0.0.1:41321)
I211103 06:13:22.248407 1383 1@server/server.go:1693  [n1] 101  starting postgres server at 127.0.0.1:33531 (use: 127.0.0.1:33531)
I211103 06:13:22.248680 1383 1@server/server.go:1695  [n1] 102  starting grpc server at 127.0.0.1:40075
I211103 06:13:22.248855 1383 1@server/server.go:1696  [n1] 103  advertising CockroachDB node at 127.0.0.1:40075
I211103 06:13:22.293626 1383 1@util/log/event_log.go:32  [n1] 104 ={"Timestamp":1635920002293613724,"EventType":"node_join","NodeID":1,"StartedAt":1635920002212568747,"LastUp":1635920002212568747}
I211103 06:13:22.294188 1383 sql/sqlliveness/slinstance/slinstance.go:296  [n1] 105  starting SQL liveness instance
I211103 06:13:22.304038 1707 sql/sqlliveness/slstorage/slstorage.go:397  [n1] 106  inserted sqlliveness session 5ed91f16ee2e41e8bf2d0978734c10d8
I211103 06:13:22.304320 1707 sql/sqlliveness/slinstance/slinstance.go:183  [n1] 107  created new SQL liveness session 5ed91f16ee2e41e8bf2d0978734c10d8
I211103 06:13:22.304495 1690 sql/temporary_schema.go:508  [n1] 108  running temporary object cleanup background job
I211103 06:13:22.305318 1695 sql/sqlstats/persistedsqlstats/provider.go:138  [n1] 109  starting sql-stats-worker with initial delay: 51m13.463828969s
I211103 06:13:22.309527 1690 sql/temporary_schema.go:552  [n1] 110  found 0 temporary schemas
I211103 06:13:22.309830 1690 sql/temporary_schema.go:555  [n1] 111  early exiting temporary schema cleaner as no temporary schemas were found
I211103 06:13:22.310088 1690 sql/temporary_schema.go:556  [n1] 112  completed temporary object cleanup job
I211103 06:13:22.310275 1690 sql/temporary_schema.go:639  [n1] 113  temporary object cleaner next scheduled to run at 2021-11-03 06:43:22.304380315 +0000 UTC
W211103 06:13:22.313827 1517 kv/range_lookup.go:243  [n1,rangefeed=table-stats-cache,range-lookup=/Meta2/Table/20/NULL] 114  range lookup of key /Meta2/Table/20/NULL found only non-matching ranges []; retrying
W211103 06:13:22.318630 1516 kv/range_lookup.go:243  [n1,rangefeed=table-stats-cache,range-lookup=/Table/20] 115  range lookup of key /Table/20 found only non-matching ranges []; retrying
I211103 06:13:22.333170 1504 kv/kvserver/replica_rangefeed.go:657  [n1,rangefeed=table-stats-cache,s1,r16/1:/Table/2{0-1}] 116  RangeFeed closed timestamp is empty
I211103 06:13:22.428691 1926 util/log/event_log.go:32  [n1,intExec=optInToDiagnosticsStatReporting] 117 ={"Timestamp":1635920002373771916,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"diagnostics.reporting.enabled\" = true","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-optInToDiagnosticsStatReporting","SettingName":"diagnostics.reporting.enabled","Value":"true"}
I211103 06:13:22.457047 2052 migration/migrationmanager/manager.go:110  [n1,intExec=set-setting,migration-mgr] 118  no need to migrate, cluster already at newest version
I211103 06:13:22.468289 2052 util/log/event_log.go:32  [n1,intExec=set-setting] 119 ={"Timestamp":1635920002447999287,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING version = $1","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-set-setting","PlaceholderValues":["'21.2-4'"],"SettingName":"version","Value":"21.2-4"}
I211103 06:13:22.571527 2076 util/log/event_log.go:32  [n1,intExec=initializeClusterSecret] 120 ={"Timestamp":1635920002548287964,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"cluster.secret\" = gen_random_uuid()::STRING","Tag":"SET CLUSTER SETTING","User":"root","ApplicationName":"$ internal-initializeClusterSecret","SettingName":"cluster.secret","Value":"048b0786-83c7-4c25-9612-8f4e2e9799cb"}
I211103 06:13:22.608860 2151 5@util/log/event_log.go:32  [n1,intExec=create-default-DB] 121 ={"Timestamp":1635920002584293035,"EventType":"create_database","Statement":"CREATE DATABASE IF NOT EXISTS defaultdb","Tag":"CREATE DATABASE","User":"root","DescriptorID":50,"ApplicationName":"$ internal-create-default-DB","DatabaseName":"defaultdb"}
I211103 06:13:22.625882 1623 3@vendor/github.com/cockroachdb/pebble/event.go:664  [n1,pebble,s1] 122  [JOB 3] WAL created 000004
I211103 06:13:22.634437 2078 5@util/log/event_log.go:32  [n1,intExec=create-default-DB] 123 ={"Timestamp":1635920002610611250,"EventType":"create_database","Statement":"CREATE DATABASE IF NOT EXISTS postgres","Tag":"CREATE DATABASE","User":"root","DescriptorID":51,"ApplicationName":"$ internal-create-default-DB","DatabaseName":"postgres"}
I211103 06:13:22.752802 1383 server/server_sql.go:1107  [n1] 124  done ensuring all necessary startup migrations have run
I211103 06:13:22.753522 2571 jobs/job_scheduler.go:357  [n1] 125  waiting 2m0s before scheduled jobs daemon start
I211103 06:13:22.767574 2573 server/auto_upgrade.go:55  [n1] 126  no need to upgrade, cluster already at the newest version
I211103 06:13:23.156907 10 util/admission/granter.go:860  [-] 127  CPULoad switching to period 250ms
I211103 06:13:23.187365 1383 1@server/server.go:2345  [n1] 128  serving sql connections
I211103 06:13:23.191829 1383 testutils/testcluster/testcluster.go:1158  [-] 129  WaitForFullReplication
I211103 06:13:23.192008 1383 testutils/testcluster/testcluster.go:1162  [-] 130  WaitForFullReplication took: 1.423µs
I211103 06:13:23.192258 1383 2@rpc/context.go:1045  [n1] 131  dialing n1: 127.0.0.1:40075 (default)
I211103 06:13:23.274327 2592 util/log/event_log.go:32  [n1,client=127.0.0.1:59458,hostssl,user=root] 132 ={"Timestamp":1635920003246045471,"EventType":"set_cluster_setting","Statement":"SET CLUSTER SETTING \"kv.protectedts.reconciliation.interval\" = '1ms'","Tag":"SET CLUSTER SETTING","User":"root","SettingName":"kv.protectedts.reconciliation.interval","Value":"00:00:00.001"}
    jobs_protected_ts_test.go:203: 
        	Error Trace:	jobs_protected_ts_test.go:203
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 0
        	Test:       	TestSchedulesProtectedTimestamp
I211103 06:13:23.580935 1383 testutils/testcluster/testcluster.go:106  [-] 133  TestCluster quiescing nodes
I211103 06:13:23.581519 2633 2@rpc/context.go:688  [n1] 134  closing [{n1: 127.0.0.1:40075 (default)} {n0: 127.0.0.1:40075 (default)}]
W211103 06:13:23.581746 2715 kv/kvserver/intentresolver/intent_resolver.go:790  [-] 135  failed to gc transaction record: could not GC completed transaction anchored at /Table/37/1/707295980303384577/0: node unavailable; try another peer
W211103 06:13:23.582786 1900 jobs/registry.go:840  [-] 136  canceling all adopted jobs due to stopper quiescing
W211103 06:13:23.584624 1707 sql/sqlliveness/slinstance/slinstance.go:220  [n1] 137  exiting heartbeat loop
W211103 06:13:23.585966 2838 kv/txn.go:693  [-] 138  failure aborting transaction: node unavailable; try another peer; abort caused by: failed to fetch protectedts metadata: failed to read metadata: protectedts-GetMetadata: context canceled
E211103 06:13:23.586971 1703 kv/kvserver/protectedts/ptreconcile/reconciler.go:151  [n1] 139  failed to refresh the protected timestamp cache to 1635920003.576280798,0: failed to fetch protectedts metadata: failed to read metadata: protectedts-GetMetadata: context canceled
I211103 06:13:23.622593 1497 kv/kvclient/kvcoord/transport_race.go:107  [-] 140  transport race promotion: ran 39 iterations on up to 327 requests
--- FAIL: TestSchedulesProtectedTimestamp (1.71s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSchedulesProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/jobs/jobsprotectedts/jobs_protected_ts_test.go
+++ b/pkg/jobs/jobsprotectedts/jobs_protected_ts_test.go
@@ -96,10 +96,13 @@ func TestJobsProtectedTimestamp(t *testing.T) {
 	_, recRemains := mkJobAndRecord()
 	ensureNotExists := func(ctx context.Context, txn *kv.Txn, ptsID uuid.UUID) (err error) {
 		_, err = ptp.GetRecord(ctx, txn, ptsID)
+		if err == nil {
+			return errors.New("found pts record, waiting for ErrNotExists")
+		}
 		if errors.Is(err, protectedts.ErrNotExists) {
 			return nil
 		}
-		return errors.NewAssertionErrorWithWrappedErrf(err, "waiting for ErrNotExists")
+		return errors.Wrap(err, "waiting for ErrNotExists")
 	}
 	testutils.SucceedsSoon(t, func() (err error) {
 		return s0.DB().Txn(ctx, func(ctx context.Context, txn *kv.Txn) error {
@@ -173,10 +176,13 @@ func TestSchedulesProtectedTimestamp(t *testing.T) {
 	_, recSchedule := mkScheduleAndRecord("do-not-drop")
 	ensureNotExists := func(ctx context.Context, txn *kv.Txn, ptsID uuid.UUID) (err error) {
 		_, err = ptp.GetRecord(ctx, txn, ptsID)
+		if err == nil {
+			return errors.New("found pts record, waiting for ErrNotExists")
+		}
 		if errors.Is(err, protectedts.ErrNotExists) {
 			return nil
 		}
-		return errors.NewAssertionErrorWithWrappedErrf(err, "waiting for ErrNotExists")
+		return errors.Wrap(err, "waiting for ErrNotExists")
 	}
 	testutils.SucceedsSoon(t, func() (err error) {
 		return s0.DB().Txn(ctx, func(ctx context.Context, txn *kv.Txn) error {

